wallet-api/src/Wallet/UTXO.hs

```diff
@@ -1,10 +1,17 @@
-{-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE LambdaCase        #-}
-{-# LANGUAGE RecordWildCards   #-}
-{-# LANGUAGE TemplateHaskell   #-}
-{-# LANGUAGE TupleSections     #-}
-{-# LANGUAGE ViewPatterns      #-}
+{-# LANGUAGE DefaultSignatures  #-}
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE FlexibleContexts   #-}
+{-# LANGUAGE FlexibleInstances  #-}
+{-# LANGUAGE LambdaCase         #-}
+{-# LANGUAGE OverloadedStrings  #-}
+{-# LANGUAGE RecordWildCards    #-}
+{-# LANGUAGE TemplateHaskell    #-}
+{-# LANGUAGE TupleSections      #-}
+{-# LANGUAGE ViewPatterns       #-}
 {-# OPTIONS -fplugin=Language.Plutus.CoreToPLC.Plugin -fplugin-opt Language.Plutus.CoreToPLC.Plugin:dont-typecheck #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Why do we need orphans here?

Reply:
This is so we can create `Serialize`, `FromJSON` and `ToJSON` for `PlcCode`

Reply:
Why not do that where it's defined? Then you also don't break the deliberate abstraction.

Reply:
i can do, I just didn't like the idea of defining JSON instances in a GHC plugin. Perhaps this type should be defined somewhere else? It doesn't really seem to be in the domain of the plugin.

Reply:
IDK, the `PlcCode` type is the abstract type that the plugin produces, we want people to be able to easily serialize it and pass it about. At the moment it has an ad hoc method for getting the serialized code, IMO it's even better to just have the appropriate typeclass instances.

So I say put them in there.

Reply:
ok done but note @vmchale that there are still orphans needed for `Digest SHA256`